.github/workflows/ci.yml

```diff
@@ -3,87 +3,57 @@ on: [push]
 
 jobs:
   pre-commit-checks:
-    name: pre-commit checks
+    name: Linux - pre-commit checks
     timeout-minutes: 30
     runs-on: ubuntu-latest
     env:
       PRE_COMMIT_USE_MICROMAMBA: 1
```

Comment:
I'm curious. I think this PR changes the way we run pre-commit hooks. Is PRE_COMMIT_USE_MICROMAMBA still necessary?

Glum also kept it, so maybe I'm misunderstanding this.

Reply:
Good point. Probably not, as pre-commit is not installing anything anymore. I'll try if it works without this option and revert if it turns out to be necessary. :)

Reply:
Seems to work, so let's get rid of it.